server/api/support.py

```diff
@@ -407,6 +407,24 @@ def log_in_as_jurisdiction_admin(email: str):
     return redirect("/")
 
 
+@api.route(
+    "/support/elections/<election_id>/login", methods=["GET"],
+)
+@restrict_access_support
```

Comment:
The other possible API was to augment the existing `/support/audit-admins/<email>/login` endpoint to take an `election_id` query param. But in that world, the client would still have to provide the email of a specific audit admin to log in as. Having the server look up an email on its own makes the endpoint much more reusable across client contexts.

Reply:
Makes sense to me!